apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -68,58 +69,57 @@ contract DisputableVoting is IForwarder, DisputableAragonApp {
-    uint64 public supportRequiredPct;
-    uint64 public minAcceptQuorumPct;
-    uint64 public voteTime;
-    uint64 public executionDelay;
-    uint64 public quietEndingPeriod;
-    uint64 public quietEndingExtension;
+    struct Vote {
+        uint256 yea;                                        // Cast vote power in favor
+        uint256 nay;                                        // Cast vote power against
+        uint256 votingPower;                                // Voting power at the moment the vote was created
+        uint256 settingId;                                  // Identification number of the setting object at the moment the vote was created
+        uint256 actionId;                                   // Identification number of the disputable action in the context of the agreement
+        VoteStatus status;                                  // Status of the disputable vote
+        uint64 startDate;                                   // Datetime when the vote was created
+        uint64 snapshotBlock;                               // Block number when the vote was created
+        uint64 pausedAt;                                    // Datetime when the vote was paused
+        uint64 pauseDuration;                               // Duration in seconds while the vote has been paused
+        uint64 quietEndingExtendedSeconds;                  // Total number of seconds a vote was extended due to quiet ending
+        bytes executionScript;                              // EVM script attached to the vote
+        mapping (address => CastVote) castVotes;            // Cast votes information indexed by voter address: status and caster address
+    }
+
+    uint64 public voteTime;                                 // Duration of each vote
+    MiniMeToken public token;                               // Token to be used for voting power, we assume it's not malicious
+
+    uint256 internal settingsLength;                        // Number of existing settings
+    mapping (uint256 => Setting) internal settings;         // List of settings indexed by ID
 
     // We are mimicking an array, we use a mapping instead to make app upgrades more graceful
-    mapping (uint256 => Vote) internal votes;
-    uint256 public votesLength;
-
-    uint64 public overruleWindow;
-    mapping (address => address) internal representatives;
+    uint256 public votesLength;                             // Number of existing votes created
+    mapping (uint256 => Vote) internal votes;               // List of votes indexed by ID
+    mapping (address => address) internal representatives;  // List of representatives indexed by voter address
 
     event StartVote(uint256 indexed voteId, address indexed creator, bytes context);
-    event CastVote(uint256 indexed voteId, address indexed voter, bool supports, uint256 stake);
+    event VoteCast(uint256 indexed voteId, address indexed voter, bool supports, uint256 stake);
```

Comment:
Why this change? It seems to be the opposite order of the other events (verb + direct object), like `StartVote, `ChangeSupportRequired`, etc.

Reply:
The problem was the struct, I can probably swap the struct then